src/drivers/dw/dma.c

```diff
@@ -282,7 +282,7 @@ struct dma_chan_data {
 	uint32_t cfg_lo;
 	uint32_t cfg_hi;
 	struct dma_id id;
-	uint32_t timer_delay;
+	uint32_t timer;
```

Comment:
Why are you renaming this? This is still referring to a time interval, right? Maybe it would be good to keep "timer_delay" to help differentiate it from the "bool timer" from struct dma_sg_config?

Reply:
It's the same.